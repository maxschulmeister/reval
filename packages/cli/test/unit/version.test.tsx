import { describe, it, expect } from 'vitest';
import React from 'react';
import { render } from 'ink-testing-library';
import Version from '../../src/commands/version';

describe('Version Command', () => {
  it('renders version information correctly', () => {
    const { lastFrame } = render(<Version />);
    
    expect(lastFrame()).toContain('reval CLI v0.1.0');
    expect(lastFrame()).toContain('core v0.1.0');
  });

  it('matches snapshot', () => {
    const { lastFrame } = render(<Version />);
    
    expect(lastFrame()).toMatchSnapshot();
  });
});