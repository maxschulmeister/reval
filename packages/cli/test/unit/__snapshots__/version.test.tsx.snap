// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Version Command > matches snapshot 1`] = `"reval CLI v0.1.0 (core v0.1.0)"`;
